index.html

```diff
@@ -285,6 +285,11 @@ <h1 class="white-header">FOSSASIA on social media</h1>
                         </div>
                     </div>
                 </div>
+				<!--Flickr-->
+				<div class="row">
+					<div id="flickrembed"></div>
+					<small style="display: block; text-align: center; margin: 0 auto;"><a href="//www.flickr.com/photos/fossasia">Fossasia on Flickr</a>.</small>
```

Comment:
Line is longer than allowed. (161 > 100)

*LineLengthBear, severity NORMAL, section `default`.*